src/main/scala/StepStatus.scala

```diff
@@ -169,6 +171,10 @@ class StepStatus(val stepNumber: String, val stepId: String, props: Properties)
     }
   }
 
+  def stepHasReduceStage(hss: HadoopStepStats): Boolean = {
+    hss.getCounterValue("org.apache.hadoop.mapreduce.JobCounter", "TOTAL_LAUNCHED_REDUCES").toLong > 0L
```

Comment:
Hmm this is the counter I pulled from the ones we use in the UI I can take another look and see, I agree we want a total instead if one is available before launch time or this will wiggle.